src/App.js

```diff
@@ -1,14 +1,18 @@
 import React from "react";
 import "./App.css";
+import Header from './head'
+import Body from './body'
+import Footer from './foot'
+import { StyledPage } from './StyledPage';
+
 
 function App() {
   return (
-    <div className="App">
-      <p>
-        Read through the instructions in the README.md file to build your NASA
-        app! Have fun 🚀!
-      </p>
-    </div>
+    <StyledPage>
```

Comment:
generally speaking, you should retrieve the data on the top level so that it can be passed down to all child components that need it.  In here your data is only needed at the Body component so it's not as important.